src/pkg/packaging/rpm/package.targets

```diff
@@ -8,14 +8,19 @@
   <PropertyGroup>
     <IsRPMBasedDistro Condition="'$(InstallerExtension)' == '.rpm'">true</IsRPMBasedDistro>
     <BuildRpmPackage Condition="'$(IsRPMBasedDistro)' == true and '$(TargetArchitecture)' == 'x64'">true</BuildRpmPackage>
+    <BuildRuntimeRpms>true</BuildRuntimeRpms>
```

Comment:
I don't see this property getting set to `false` in the buildpipeline\Core-Setup-Linux-BT.json file.  For .deb packaging, I see that we set it to false during the `Package Runtime Dep` steps.

Reply:
That is really imported. I set BuildRuntimeRpms to false for all rpm package step except the first one.